packages/payment-plugin/package.json

```diff
@@ -1,30 +1,36 @@
 {
-  "name": "@vendure/payments-plugin",
-  "version": "1.3.2",
-  "license": "MIT",
-  "main": "package/index.js",
-  "types": "package/index.d.ts",
-  "files": [
-    "package/**/*"
-  ],
-  "private": false,
-  "scripts": {
-    "watch": "tsc -p ./tsconfig.build.json --watch",
-    "build": "rimraf package && tsc -p ./tsconfig.build.json",
-    "lint": "tslint --fix --project ./",
-    "ci": "yarn build"
-  },
-  "homepage": "https://www.vendure.io/",
-  "funding": "https://github.com/sponsors/michaelbromley",
-  "publishConfig": {
-    "access": "public"
-  },
-  "devDependencies": {
-    "@types/braintree": "^2.22.15",
-    "@vendure/common": "^1.3.2",
-    "@vendure/core": "^1.3.2",
-    "braintree": "^3.0.0",
-    "rimraf": "^3.0.2",
-    "typescript": "4.3.5"
-  }
+    "name": "@vendure/payments-plugin",
+    "version": "1.3.2",
+    "license": "MIT",
+    "main": "package/index.js",
+    "types": "package/index.d.ts",
+    "files": [
+        "package/**/*"
+    ],
+    "private": false,
+    "scripts": {
+        "watch": "tsc -p ./tsconfig.build.json --watch",
+        "build": "rimraf package && tsc -p ./tsconfig.build.json",
+        "e2e": "jest --config ../../e2e-common/jest-config.js --runInBand --package=payment-plugin",
+        "lint": "tslint --fix --project ./",
+        "ci": "yarn build"
+    },
+    "homepage": "https://www.vendure.io/",
+    "funding": "https://github.com/sponsors/michaelbromley",
+    "publishConfig": {
+        "access": "public"
+    },
+    "dependencies": {
+        "@mollie/api-client": "^3.5.1"
+    },
+    "devDependencies": {
+        "@types/braintree": "^2.22.15",
+        "@vendure/common": "^1.3.2",
+        "@vendure/core": "^1.3.2",
+        "@vendure/testing": "^1.3.2",
+        "braintree": "^3.0.0",
```

Comment:
Why is braintree a dev dependency? Should mollie be a devdependency as well?

Reply:
Yes, make it a dev dependency please. It is this way because if I want to install the payment plugin but only want to use Mollie, I don't also want to install the Braintree (Stripe, Adyen etc) packages too. The docs need to include the instruction to install the required package manually.

Reply:
Makes sense. I've moved Mollie to devDependencies, and added peerDependencies for both Braintree and Mollie with their current major version: `3.x`